src/Microsoft.AspNetCore.SignalR.Core/DefaultHubLifetimeManager.cs

```diff
@@ -123,17 +113,20 @@ public override Task InvokeGroupAsync(string groupName, string methodName, objec
                 throw new ArgumentNullException(nameof(groupName));
             }
 
-            return InvokeAllWhere(methodName, args, connection =>
+            List<Task> tasks = new List<Task>();
+            InvocationMessage message = CreateInvocationMessage(methodName, args);
```

Comment:
Move this inside the below `if` check so we don't create a message when there is no group.